avr/cores/megacommand/Midi/MidiSysex.hh

```diff
@@ -69,18 +69,50 @@ protected:
   bool recording;
   uint8_t recvIds[3];
   bool sysexLongId;
+  uint8_t *data;
+  uint8_t *recordBuf;
+  volatile uint8_t *sysex_highmem_buf;
 
 public:
   void startRecord(uint8_t *buf = NULL, uint16_t maxLen = 0);
```

Comment:
I see `maxLen` is stored to `maxRecordLen` no matter it's a supplied buf or `sysex_highmem_buf` -- we then have a few `if (n < maxRecordLen)` checks..